platform/docs/docs/configuration/configurationFiles.md

```diff
@@ -111,13 +111,15 @@ window.config = ({ servicesManager } = {}) => {
 ## Configuration Options
 
 Here are a list of some options available:
+- `minDisplaySetsToRunHP`:  If configured, sets the minimum display sets needed to run the hanging protocol and start displaying images. Used when OHIF needs to handle studies with several series and it is required that the first image should be loaded faster. Only changed if really necessary.
```

Comment:
what is `  panel and locking already stored measurements.`? 

Please also explain that HP migh break if you chose to run without the full metadata